### ISSUE TITLE
add metadata on circuit vs application function to raw_data

### ISSUE THREAD

**Issue by johannesgreiner** [Qiskit/qiskit-ibm-catalog#41]
### What is the expected enhancement?

For CI/CD, it'd be nice to add to the` raw_data` dict the simple info if it's a Circuit function or Application function. Since we can assume a more standardized I/O for circuit functions

**Comment by Tansito** [comment]
I will need more context for this, @jogreiner. From what I'm understanding, what you want is to receive if the function is an Application or a Circuit Function, don't you? Do you have any other use-case in mind?

**Comment by johannesgreiner** [comment]
yes this is correct. 

I am focused on the usecase of writing e2e tests specific to both types. For Circuit Functions I need to pass (a library of) circuits. I can either hard-code a list and aim to have this updated with each new Function release (can be up to ~10/year) or have it automated with this feature. 

To be creative about other use-cases, as a users with access to several Functions, I might want to filter programmatically between Circuit Functions and Application Functions as well. E.g. I could compare/benchmark different Circuit Functions.

**Comment by Tansito** [comment]
Yep, sounds reasonable to me. The first point I think is easy to do, let me prioritize it but I think we can be able to do it for the next release 0.20.0. I will think in the solution these days and give you a proposal.

The second one should be easy too but it will require a bit more of work so I'm not sure if I'm going to be able to prioritize it accordingly for the next version. Anyway, let me sync with @pandasa123 and keep an eye on this, @johannesgreiner . I think it has a lot of sense.

**Comment by johannesgreiner** [comment]
@Tansito did the circuit function vs application function metadata field get some love lately?

**Comment by Tansito** [comment]
Sorry @johannesgreiner , I couldn't answer you earlier. It was impossible for us to address it for versions 0.20.0 and 0.21.0. I'm really sorry. I will prioritize it for 0.22.0 after the deployment of the integration with IBM Cloud 🙏

**Comment by Tansito** [comment]
@johannesgreiner I included the type of the Function in the last release? Let us know if you would need something more from this issue 🙏
